cake/console/cake.php

```diff
@@ -371,7 +371,7 @@ function dispatch() {
 			$methods = array_diff(get_class_methods('Shell'), array('help'));
 		}
 		$methods = array_diff(get_class_methods($Shell), $methods);
-		$added = in_array(strtolower($arg), array_map('strtolower', $methods));
+		$added = $arg && in_array(strtolower($arg), array_map('strtolower', $methods));
```

Comment:
This has possibly the same problem (and solution) as in the `Inflector` below: `'0'` is interpreted as `false`, but shouldn't.